src/features/testing/assistant.md

```diff
@@ -1 +1,38 @@
-# Assistant (WIP)
+# Assistant
+
+> **PRO FEATURE**
+
+The Assistant page is a feature offered for **Pro users**.
+It helps you understand requests, explains elements like headers and suggests attack vectors.
+
+The AI assistant in Caido is taylored for security research.
+
+Data is sent to a third party (OpenAI) and can be stored for up to 30 days.
```

Comment:
Make is a quote (the `>` thing) and lets link to our privacy policy and maybe their privacy policy?